### PR TITLE
Update libG for 2.10 RC

### PR DESCRIPTION
**Pull request by mjkkirschner**
### Purpose

bring in latest libG nugets fixing some crash bugs.
I will list change log here.

changelog:

* fix a crash when creating an arc or ellipse when doing a projection to a plane
* improve roboustness of parameter at point on curves

### Declarations

Check these if you believe they are true

- [ ] The codebase is in a better state after this PR
- [ ] Is documented according to the [standards](https://github.com/DynamoDS/Dynamo/wiki/Coding-Standards)
- [ ] The level of testing this PR includes is appropriate
- [ ] User facing strings, if any, are extracted into `*.resx` files
- [ ] All tests pass using the self-service CI.
- [ ] Snapshot of UI changes, if any.
- [ ] Changes to the API follow [Semantic Versioning](https://github.com/DynamoDS/Dynamo/wiki/Dynamo-Versions) and are documented in the [API Changes](https://github.com/DynamoDS/Dynamo/wiki/API-Changes) document.
- [ ] This PR modifies some build requirements and the readme is updated

### CODE REVIEW

**Comment by aparajit-pratap** [comment]
Do we not want to wait for the LibG build fixes?

**Comment by mjkkirschner** [comment]
@aparajit-pratap - sure, but should it matter? Because the actual builds here are done with a specified version of the compilers there should be no issues.

**Comment by aparajit-pratap** [comment]
> @aparajit-pratap - sure, but should it matter? Because the actual builds here are done with a specified version of the compilers there should be no issues.

I suppose so.

File: src/Config/Cleanup.targets [hunk 1]
```diff
@@ -8,6 +8,7 @@
       <FilesToDelete Include="$(OutputPath)nodes\SharpDX*"/>
       <FilesToDelete Include="$(OutputPath)nodes\HelixToolkit*"/>
       <FilesToDelete Include="$(OutputPath)nodes\Cyotek*"/>
+      <FilesToDelete Include="$(OutputPath)nodes\ProtoGeometry*"/>
```
Comment (aparajit-pratap) [inline src/Config/Cleanup.targets:11]: 👍

**Review by aparajit-pratap — APPROVED** [review]

File: src/Config/Cleanup.targets (hunk 1 above)
Comment (mjkkirschner) [reply src/Config/Cleanup.targets:11]: so one thing I forgot to mention about this - is that these are not foolproof - these targets are only imported into a few projects, so if another project that builds after copies the specified binaries *again* we'll have the same trouble - we really should have a way to add a target for post build to the entire .sln - but I think thats not straight forward.

Comment (mjkkirschner) [reply src/Config/Cleanup.targets:11]: alternatively, we could import them into all projects as part of our standard import I guess.

**Comment by mjkkirschner** [comment]
cherry picking to 2.10

**Pull request merged**